src/content/docs/de/core-concepts/sharing-state.mdx

```diff
@@ -0,0 +1,694 @@
+- **Sie sind Framework-agnostisch** Das bedeutet, dass die gemeinsame Nutzung von Zuständen zwischen Frameworks nahtlos ist! Astro ist auf Flexibilität ausgelegt, daher lieben wir Lösungen, die unabhängig von deiner Präferenz eine ähnliche Entwicklererfahrung bieten.
+
+Dennoch gibt es eine Reihe von Alternativen, die du ausprobieren kannst. Dazu gehören:
+- [Svelte's eingebaute Stores](https://svelte.dev/tutorial/writable-stores)
+- [Solide signals](https://www.solidjs.com/docs/latest) außerhalb eines Komponentenkontextes
+- [Vue's Reaktivitäts-API](https://vuejs.org/guide/scaling-up/state-management.html#simple-state-management-with-reactivity-api)
+- [Senden von benutzerdefinierten Browser-Ereignissen](https://developer.mozilla.org/en-US/docs/Web/Events/Creating_and_triggering_events) zwischen Komponenten
+
+:::note[FAQ]
+
+<details>
+<summary>**🙋 Kann ich Nano Stores in `.astro`-Dateien oder anderen serverseitigen Komponenten verwenden?**</summary>
+
+Nano Stores _können_ in serverseitigen Komponenten importiert, beschrieben und gelesen werden, **aber wir empfehlen es nicht!** Das liegt an einigen Einschränkungen:
+- Das Schreiben in einen Store aus einer `.astro`-Datei oder einer [nicht-hydratatierten Komponente](/de/core-concepts/framework-components/#interaktive-komponenten-hydratisieren) hat _keine_ Auswirkung auf den Wert, der von [Client-seitigen Komponenten](/de/reference/directives-reference/#client-directives) empfangen wird.
+- Du kannst einen Nano Store nicht als "prop" an clientseitige Komponenten übergeben.
+- Du kannst keine Änderungen aus einer `Astro`-Datei abonnieren, da Astro-Komponenten nicht neu gerendert werden.
+
+Wenn du diese Einschränkungen verstehst und trotzdem einen Anwendungsfall findest, kannst du Nano Stores ausprobieren! Denke daran, dass Nano Stores speziell für die Reaktivität auf Änderungen auf dem **Client** konzipiert sind.
+
+</details>
+
+<details>
+<summary>**🙋 Wie unterscheiden sich die Svelte Stores von den Nano Stores?**</summary>
+
+**Nano Stores und [Svelte Stores](https://svelte.dev/tutorial/writable-stores) sind sich sehr ähnlich!** Tatsächlich kannst du in [Nanostores die gleiche `$`-Verknüpfung](https://github.com/nanostores/nanostores#svelte) für Subscriptions verwenden, die du auch in Svelte Stores benutzen kannst.
+
+Wenn du Bibliotheken von Drittanbietern vermeiden willst, sind [Svelte Stores](https://svelte.dev/tutorial/writable-stores) ein großartiges Werkzeug für die inselübergreifende Kommunikation. Dennoch könntest du Nano Stores bevorzugen, wenn dir a) die Add-ons für ["objects"](https://github.com/nanostores/nanostores#maps) und [async state](https://github.com/nanostores/nanostores#lazy-stores) gefallen oder b) du zwischen Svelte und anderen UI-Frameworks wie Preact oder Vue kommunizieren möchtest.
+</details>
+
+<details>
+<summary>**🙋 Wie unterscheiden sich Feststoffsignale von Nano Stores?**</summary>
```

Comment:
```suggestion
<summary>**🙋 Wie unterscheiden sich Solid Signals von Nano Stores?**</summary>
```

Ist ein feststehender Begriff aus der Welt von Solid JS.

Reply:
Sure. Thank you Co-Pilot 😉